src/matrix3.js

```diff
@@ -19,24 +19,21 @@
 // THE SOFTWARE.
 
 import MathArray from './lib/math-array';
-// import {checkNumber} from './lib/common';
+import {checkNumber} from './lib/common';
 import Vector2 from './vector2';
 import Vector3 from './vector3';
-// import Vector2, {validateVector2} from './vector2';
-// import Vector3, {validateVector3} from './vector3';
-// import assert from 'assert';
+import {validateVector} from './matrix4';
```

Comment:
We should minimize imports between similar classes. Either copy this function or move it to a utility file.

Reply:
`validateVector` is moved to a new file `src/lib/validators.js`